geometry_msgs/PointStamped start_point
float64 wrist_yaw
geometry_msgs/Pose2D goal_pose
bool spin_to_heading
string push_controller
string perceptual_proxy
string action_primitive
bool left_arm
bool right_arm
bool high_arm_init
bool open_gripper
---
geometry_msgs/Pose2D goal_error
bool action_aborted